scripts-dev/check_schema_delta.py

```diff
@@ -30,7 +38,7 @@ def main(force_colors: bool) -> None:
     click.secho("Updating repo...")
 
     repo = git.Repo()
-    repo.remote().fetch()
+    repo.remote().fetch(refspec="develop")
```

Comment:
Label this magic value

```suggestion
    repo.remote().fetch(refspec=DEFAULT_BRANCH)
```

Did this spawn from something you saw? I assume GitHub would give you the default branch anyway if you don't specify one?

Reply:
This was purely to speed things up, so that we only pull that one branch that we care about rather than all branches